src/main/java/core/basesyntax/service/operations/ReturnOperationsHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Operations;
```

Comment:
The import statement for 'core.basesyntax.model.Operations' is not used in this class. If it's not needed, it's better to remove it to keep the code clean.